Gopkg.lock

```diff
@@ -1,1314 +0,0 @@
-# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
```

Comment:
This doesn't seem right - we want to have a gopkg.lock file

Reply:
Fixed